Assets/Scripts/PlayerController.cs

```diff
@@ -43,6 +43,9 @@ public enum PlayerType { Light, Shadow };
     // The script from the interactable object
     private Interactable interactableScript = null;
 
+    // For debugging purposes
+    public bool invincible = false;
```

Comment:
Better way would be separating out the dying mechanism to another component. That way, we can just disable or enable those components containing dying mechanism to debug and test.